spec/components/restructured_work_history/review_component_spec.rb

```diff
@@ -53,6 +55,16 @@
     )
   end
 
+  context 'when the application is incomplete' do
+    context 'when we click from the application review page' do
```

Comment:
From this component's POV, it doesn't know when it's going to be passed `return_to_application_review`. On the basis that specs should operate at the same level of abstraction as the classes they're testing, I think it's ok to just have a context of `when return_to_application_review is true`.

Reply:
makes sense, thanks!